src/main/resources/META-INF/rust-core.xml

```diff
@@ -524,6 +524,11 @@
                          enabledByDefault="true" level="ERROR"
                          implementationClass="org.rust.ide.inspections.RsTraitImplementationInspection"/>
 
+        <localInspection language="Rust" groupName="Rust"
+                         displayName="Cast to bool"
+                         enabledByDefault="true" level="ERROR"
+                         implementationClass="org.rust.ide.inspections.RsCastToBoolInspection"/>
+
         <localInspection language="Rust" groupPath="Rust" groupName="Lints"
```

Comment:
Is there an ordering to be followed in this file? I tossed the entry next to the intent for E0050 (as this PR is for E0054, so they line up).